script.js

```diff
@@ -105,7 +105,7 @@ function updateDisplay(list) {
             deleteButtonEl.setAttribute('type', 'button');
             deleteButtonEl.classList.add('deleteBtn')
             deleteButtonEl.setAttribute('value', '—');
-
+            deleteButtonEl.ariaLabel = "Delete item";
```

Comment:
Unfortunately, the .ariaLabel method doesn't work in Firefox: https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaLabel 😞 

This is a bit longer but should be more compatible: ``deleteButtonEl.setAttribute('aria-label', 'Delete item');``

Reply:
Okay, I changed it! :)